crates/node/core/build.rs

```diff
@@ -22,10 +22,68 @@ fn main() -> Result<(), Box<dyn Error>> {
     // if not on a tag: v0.2.0-beta.3-82-g1939939b
     // if on a tag: v0.2.0-beta.3
     let not_on_tag = env::var("VERGEN_GIT_DESCRIBE")?.ends_with(&format!("-g{sha_short}"));
-    let is_dev = is_dirty || not_on_tag;
-    println!("cargo:rustc-env=RETH_VERSION_SUFFIX={}", if is_dev { "-dev" } else { "" });
+    let version_suffix = if is_dirty || not_on_tag { "-dev" } else { "" };
```

Comment:
unrelated, I noticed that some docker builds have a -dev suffix, which doesn't make sense, so I think this is kinda wrong

wouldn't be opposed to removing this entirely @onbjerg